app/src/main/java/ar/com/wolox/android/example/ui/example/home/HomeFragment.kt

```diff
@@ -0,0 +1,42 @@
+package ar.com.wolox.android.example.ui.example.home
+
+import ar.com.wolox.android.R
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import kotlinx.android.synthetic.main.fragment_home.*
+import javax.inject.Inject
+
+/**
+ *
+ */
+class HomeFragment : WolmoFragment<HomePresenter>(), IHomeView {
+
+    @Inject internal lateinit var homeNewsFragment: HomeNewsFragment
+    @Inject internal lateinit var homeProfileFragment: HomeProfileFragment
+
+    private lateinit var fragmentPagerAdapter: SimpleFragmentPagerAdapter
+
+    companion object {
```

Comment:
Move companion object to the end of the file

Reply:
✔ **Done**